pyfr/plugins/fluidforce.py

```diff
@@ -105,17 +113,11 @@ def __init__(self, intg, cfgsect, suffix):
                     # Product to give J^-T at the solution points
                     rcpjact[etype] = smat*rcpdjac
 
-                # Phyiscal normals
-                pnorms = eles.get_pnorms(eidx, fidx)
-
-                eidxs[etype, fidx].append(eidx)
-                norms[etype, fidx].append(pnorms)
-
                 # Get the flux points position of the given face and element
                 # indices relative to the moment origin
                 if self._mcomp:
-                    fpts_idx = eles.basis.facefpts[fidx]
-                    rfpt = eles.plocfpts[fpts_idx, eidx] - morigin
+                    ploc = eles.plocs_at(ppts)[:,[eidx]]
```

Comment:
See if we can go without the [].  Also, spacing,

Reply:
Sadly not, I tried initially as this is ugly.